experimental/packages/opentelemetry-instrumentation-grpc/src/grpc/index.ts

```diff
@@ -292,9 +297,15 @@ export class GrpcNativeInstrumentation extends InstrumentationBase<
         )}`;
         const args = Array.prototype.slice.call(arguments);
         const metadata = getMetadata(grpcClient, original, args);
+        const { service, method } = _extractMethodAndService(original.path)
         const span = instrumentation.tracer.startSpan(name, {
           kind: SpanKind.CLIENT,
-        });
+        })
+          .setAttributes({
+            [AttributeNames.RPC_SYSTEM]: 'grpc',
```

Comment:
It can be better to extract these hardcoded literal `'grpc'` strings  to a constant variable.